airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/CdcPostgresSourceTest.java

```diff
@@ -438,4 +449,102 @@ public SourceRecord sourceRecord() {
     assertNull(pctp.getHeartbeatPosition(null));
   }
 
+  @Test
+  protected void syncShouldIncrementLSN() throws Exception {
```

Comment:
nit : Can you add comments on the different scenarios being tested?
e.g. 

1. Assert LSNs don't change before/after sync in ack after sync mode. 
2. Assert LSNs change after a subsequent with new state
...etc